wp-highlight-user.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Plugin Name: WP Hightlight User
+ * Plugin URI: http://bonnierpublications.com
+ * Description: adds a checkbox to all users to have the option highlight them
+ * Version: 0.1
+ * Author: Bonnier
+ * Author URI: http://bonnierpublications.com
+ */
+
+namespace WpHighlightUser;
+
+
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+// Handle autoload so we can use namespaces
+spl_autoload_register(function ($className) {
+
+    if (strpos($className, __NAMESPACE__) !== false) {
+        $className = str_replace("\\", DIRECTORY_SEPARATOR, $className);
+        //require_once(__DIR__ . DIRECTORY_SEPARATOR . $className . '.php');
+        require_once(__DIR__ . DIRECTORY_SEPARATOR . Plugin::CLASS_DIR . DIRECTORY_SEPARATOR . $className . '.php');
+    }
+});
+
+class Plugin
+{
+    /**
+     * Text domain for translators
+     */
+    const TEXT_DOMAIN = 'wp-highlight-user';
+    const CLASS_DIR = 'src';
+
+    /**
+     * @var object Instance of this class.
+     */
+    private static $instance;
+
+    private $settings;
+
+    /**
+     * @var string Filename of this class.
+     */
+    public $file;
+
+    /**
+     * @var string Basename of this class.
+     */
+    public $basename;
+
+    /**
+     * @var string Plugins directory for this plugin.
+     */
+    public $pluginDir;
+
+    /**
+     * @var Object
+     */
+    public $scripts;
+
+    /**
+     * @var string Plugins url for this plugin.
+     */
+    public $pluginUrl;
+
+    /**
+     * Do not load this more than once.
+     */
+    private function __construct()
+    {
+        // Set plugin file variables
+        $this->file = __FILE__;
+        $this->basename = plugin_basename($this->file);
+        $this->pluginDir = plugin_dir_path($this->file);
+        $this->pluginUrl = plugin_dir_url($this->file);
+        // Load textdomain
+        load_plugin_textdomain(self::TEXT_DOMAIN, false, dirname($this->basename) . '/languages');
+
+        new Settings\User();
+    }
+
+    /**
+     * Returns the instance of this class.
+     */
+    public static function instance()
+    {
+        if (!self::$instance) {
+            self::$instance = new self;
+            global $wpHighlightUser;
+            $wpHighlightUser = self::$instance;
+
+            /**
+             * Run after the plugin has been loaded.
+             */
+            do_action('wp_highlight_user_loaded');
+        }
+
+        return self::$instance;
+    }
+
+    public function settings()
```

Comment:
I think you don't need this function, since your registering the ACF stuff in ur constructor . ALso you can remove the $settings property.